primitives/src/lib.rs

```diff
@@ -553,6 +559,74 @@ impl TryFrom<(&str, AssetIssuer)> for CurrencyId {
 	}
 }
 
+
+pub type ZenlinkAssetId = zenlink_protocol::AssetId;
+const LP_DISCRIMINANT: u64 = 6u64;
+const PARA_CHAIN_ID: u32 = 1000;
+
+impl TryFrom<CurrencyId> for ZenlinkAssetId {
```

Comment:
This needs to be changed to something like [this](https://github.com/bifrost-finance/bifrost/blob/0eb697cc517daf9103982a6be80d6e7ea8f7e458/node/primitives/src/currency.rs#LL78C8-L78C22)?